### PR TITLE
Wpf pull modify book

### PR DESCRIPTION
**Pull request by MESOLONELY**

### CODE REVIEW

File: demos/SampleWPF/Converter/BooleanToVisibilityConverter.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace AiurVersionControl.SampleWPF.Converter
+{
+    /// <summary>
+    /// Converts a Boolean into a Visibility.
+    /// </summary>
+    public class BooleanToVisibilityConverter : IValueConverter
```
Comment (Anduin2017) [inline demos/SampleWPF/Converter/BooleanToVisibilityConverter.cs]: I guess wpf already provided a tool like this. See how I display the server address bar.

**Review by Anduin2017 — CHANGES_REQUESTED** [review]

File: demos/SampleWPF/Converter/BooleanToVisibilityConverter.cs (hunk 1 above)
Comment (MESOLONELY) [reply demos/SampleWPF/Converter/BooleanToVisibilityConverter.cs]: Wpf's 'BooleanToVisibilityConverter' seems complicated to convert 'False -> visible', so i add an isConvert parameter in it.

**Review by Anduin2017 — APPROVED** [review]

File: demos/SampleWPF/Components/BookListItem/BookListItem.xaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <Grid Margin="0,2" x:Name="BookGrid" >
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="20" />
+                <ColumnDefinition Width="20" />
+                <ColumnDefinition Width="140" />
+                <ColumnDefinition Width="160" />
+            </Grid.ColumnDefinitions>
+            <Button x:Name="DeleteButton" Grid.Column="0" Content="🗑️" Visibility="Visible" BorderBrush="Transparent" 
+                    Background="Transparent" Command="{Binding CommitDrop}"/>
+            <Button x:Name="EditButton" Grid.Column="1" Content="✏️" Visibility="Visible" BorderBrush="Transparent"
```
Comment (Anduin2017) [inline demos/SampleWPF/Components/BookListItem/BookListItem.xaml]: The visibility is useless right?

File: demos/SampleWPF/Windows/MainWindow.xaml [hunk 3]
```diff
@@ -7,7 +7,7 @@
         mc:Ignorable="d"
         xmlns:windows="clr-namespace:AiurVersionControl.SampleWPF.Windows"
         xmlns:cp="clr-namespace:AiurVersionControl.SampleWPF.Components"
-        Title="VersionControlled CRUD Demo App"
+        Title="{Binding CrudPresenter.NewTitle, StringFormat=VersionControlled CRUD Demo App - {0}}"
```
Comment (Anduin2017) [inline demos/SampleWPF/Windows/MainWindow.xaml]: What's that for?

**Pull request merged**